python/selfie-lib/tests/RecordCall_test.py

```diff
@@ -0,0 +1,34 @@
+from unittest.mock import Mock
+from selfie_lib.WriteTracker import CallLocation, CallStack, recordCall
+
+
+def test_call_location_ide_link():
+    layout = Mock()
+    location = CallLocation(file_name="example.py", line=10)
+    expected_link = "File: example.py, Line: 10"
+
+    assert location.ide_link(layout) == expected_link
+
+
+def test_call_stack_ide_link():
+    layout = Mock()
+    location1 = CallLocation(file_name="example1.py", line=10)
+    location2 = CallLocation(file_name="example2.py", line=20)
+    call_stack = CallStack(location=location1, rest_of_stack=[location2])
+
+    expected_links = "File: example1.py, Line: 10\nFile: example2.py, Line: 20"
+    assert call_stack.ide_link(layout) == expected_links
+
+
+def test_record_call_with_caller_file_only_false():
+    call_stack = recordCall(False)
```

Comment:
Since this is getting called on line 24, add an assertion that the call location is `RecordCall_test L24`. For example, from the JVM test:

https://github.com/diffplug/selfie-python-wip/blob/2aededc364606e8dcd3c3f69669cb415a0b34b2f/jvm/selfie-lib/src/jvmTest/kotlin/testpkg/RecordCallTest.kt#L28-L30

https://github.com/diffplug/selfie-python-wip/blob/2aededc364606e8dcd3c3f69669cb415a0b34b2f/jvm/selfie-lib/src/jvmTest/kotlin/testpkg/RecordCallTest.kt#L45-L46

It might take some filtering to achieve this:

https://github.com/diffplug/selfie-python-wip/blob/2aededc364606e8dcd3c3f69669cb415a0b34b2f/jvm/selfie-lib/src/jvmMain/kotlin/com/diffplug/selfie/guts/WriteTracker.jvm.kt#L57-L58